### PR TITLE
added: axios in axios branch

### PR DESCRIPTION
**Pull request by Aselshine**
Issue #:
[Trello](https://trello.com/invite/b/AklPlVwg/ATTI9fdb094d2c14954c792b9e3930d9949a11CEF7C2/solid-practicum-dominos-team-2)

## PR type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Other (please describe):

## Does this introduce a breaking change?

- [ ] Yes (please describe):
- [x] No

## Pull request checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Accessibility support has been tested
- [ ] Responsive layout and functionality have been tested
  - [ ] desktop and mobile view responsiveness are supported
  - [ ] no loss of functionality or content when zoomed in to 400%
  - [ ] no horizontal scrolling when zoomed in to 400% (only vertical scrolling, exception: DataGrids)
- [ ] avoids using many levels of prop drilling
- [ ] check console on errors and warnings.
---

References:
- [Clean code](https://github.com/airbnb/javascript)

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -0,0 +1,38 @@
+.App {
```
Comment (Esenushka) [inline src/App.css]: Delete this css because we using  scss

File: src/App.js [hunk 2]
```diff
@@ -1,9 +1,23 @@
-import "./main.sass"
-
+import logo from './logo.svg';
+import './App.css';
```
Comment (Esenushka) [inline src/App.js]: change import to main.scss

File: src/App.js [hunk 3]
```diff
@@ -1,9 +1,23 @@
...
 
 function App() {
   return (
     <div className="App">
```
Comment (Esenushka) [inline src/App.js]: delete inside of div because we dont need that

File: src/App.js [hunk 4]
```diff
@@ -1,23 +1,10 @@
-import logo from './logo.svg';
```
Comment (Esenushka) [inline src/App.js]: where is import to main.sass ,if you dont have main.sass you need to get the last changes (git pull origin develop)

**Review by Esenushka — APPROVED** [review]

**Comment by Esenushka** [comment]
resolve the conflicts before merge

File: src/App.js (hunk 2 above)
Comment (Aselshine) [reply src/App.js]: Done

File: src/App.js (hunk 3 above)
Comment (Aselshine) [reply src/App.js]: Done

File: src/App.js (hunk 4 above)
Comment (Aselshine) [reply src/App.js]: Corrected

File: src/App.css (hunk 1 above)
Comment (Aselshine) [reply src/App.css]: Deleted

File: src/App.js [hunk 5]
```diff
@@ -1,14 +1,10 @@
 
 
-import './App.css';
-
-import "./main.sass"
```
Comment (Esenushka) [inline src/App.js:5]: why did you remove import main?

Comment (Aselshine) [reply src/App.js:5]: I didnt delete main
Снимок

**Comment by Esenushka** [comment]
okay i think you can merge

**Review by luchok22 — APPROVED** [review]
